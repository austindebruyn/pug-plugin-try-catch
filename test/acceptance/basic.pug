.foo
  try
    h1 alabama
  catch
    h1 australia
