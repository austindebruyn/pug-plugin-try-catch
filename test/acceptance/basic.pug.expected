<div class="foo"><h1>alabama</h1></div>
